.github/workflows/tests.yml

```
@@ -60,6 +60,11 @@ jobs:
         id: dataset-cache
       - run: python -m mne_bids_pipeline._download ${{ matrix.dataset }}
         if: steps.dataset-cache.outputs.cache-hit != 'true'
+      - run: |
+          rm ~/mne_data/ds003392/sub-01/meg/sub-01_acq-calibration_meg.dat
+          rm ~/mne_data/ds003392/sub-01/meg/sub-01_acq-crosstalk_meg.fif
```

Just realized we'll need a `-f` here because we cache the dataset. The next time it goes stale and re-caches, the next run the file will already be missing
```suggestion
          rm -f ~/mne_data/ds003392/sub-01/meg/sub-01_acq-calibration_meg.dat
          rm -f ~/mne_data/ds003392/sub-01/meg/sub-01_acq-crosstalk_meg.fif
```